ui/src/telegrafs/components/TelegrafConfigOverlay.tsx

```diff
@@ -69,18 +56,22 @@ export class TelegrafConfigOverlay extends PureComponent<Props> {
     )
   }
 
+  private handleDismiss = () => {
+    const {
+      router,
+      params: {orgID},
+    } = this.props
+
+    router.push(`/orgs/${orgID}/telegrafs`)
+  }
+
   private handleDownloadConfig = async () => {
-    try {
-      const config = await client.telegrafConfigs.getTOML(
-        this.props.telegrafConfigID
-      )
-      downloadTextFile(
-        config,
-        `${this.props.telegrafConfigName || 'config'}.toml`
-      )
-    } catch (error) {
-      this.props.notify(getTelegrafConfigFailed())
-    }
+    const {
+      onDownloadTelegrafConfig,
+      telegrafConfigName,
+      telegrafConfigID,
+    } = this.props
+    onDownloadTelegrafConfig(telegrafConfigID, telegrafConfigName)
```

Comment:
could the config id from the url be used as the config id rather than getting one from redux?